File: POOJAVA/src/clasesPOO/TipoDocumento.java
package clasesPOO;

public class TipoDocumento 
{
    String idTipoDoc;
    String nTipoDoc;
    
    public TipoDocumento(String idTipo,String nTipo)
    {
        establecerDatos(idTipo,nTipo);
    }
    public String ObtenerTipoDoc()
    {
       return idTipoDoc;
    }
    public String ObtenernTipoDoc()
    {
       return nTipoDoc;
    }
    void  establecerDatos(String idTipoD,String nTipoD)
    {
        idTipoDoc=idTipoD;
        nTipoDoc=nTipoD;
    }
}

File: POOPHP/app/Model/TipoDocumento.php
<?php
namespace App\Model;
class TipoDocumento
{
    public $idTipoDoc="";
    public $nTipoDoc="";
   public function __construct(String $idTipo,String $nTipo)
    {
        $this->establecerDatos($idTipo,$nTipo);
    }
    public function ObtenerTipoDoc()
    {
        return $this->idTipoDoc;
    }
    public function ObtenernTipoDoc()
    {
        return $this->nTipoDoc;
    }
    public function establecerDatos(String $idTipoD,String $nTipoD)
    {
        $this->idTipoDoc=$idTipoD;
        $this->nTipoDoc=$nTipoD;
    }
}
?>